controller/testController.go

```
@@ -58,10 +68,62 @@ func (c *TestController) test(ctx *gin.Context) {
 // @Accept json
 // @Produce json
 // @Success 200
-// @Router /test/ [post]
+// @Router /test/ [put]
 func (c *TestController) insert(ctx *gin.Context) {
 	t := model.Tmodel{Name: "Test insert", Uuid: uuid.New()}
-	c.db.Create(&t)
+	tmodel, err := c.testService.Create(&t)
+	if err != nil {
+		ctx.AbortWithError(http.StatusInternalServerError, err)
```

golangci-lint (1.64.8)

Error return value of `ctx.AbortWithError` is not checked

(errcheck)